ogr/services/gitlab/issue.py

```diff
@@ -102,21 +102,30 @@ def create(
         labels: Optional[List[str]] = None,
         assignees: Optional[List[str]] = None,
     ) -> "Issue":
+        assignee_ids = []
+        for user in assignees or []:
+            users_list = project.service.gitlab_instance.users.list(username=user)
+
+            if not users_list:
+                raise GitlabAPIException(f"Unable to find '{user}' username")
+
+            assignee_ids.append(str(users_list[0].id))
+
         data = {"title": title, "description": body}
         if labels:
             data["labels"] = ",".join(labels)
         if assignees:
-            data["assignee_ids"] = ",".join(assignees)
+            data["assignee_ids"] = ",".join(assignee_ids)
```

Comment:
Can we have it in a separate commit, please?

Reply:
Sure, no problem.

Reply:
Sorry that I haven't made it clear. I meant only the fix for assignees. (So it can be found in the history that we've changed something.)

Reply:
Aha, ok this is just a refactor. Not a fix.

Reply:
Yeah, I just moved the code from the project to class where it should be :D But I think it's nice to have it separate, since it's bigger change than just removing code and decorating